src/Serializer/AbstractItemNormalizer.php

```diff
@@ -387,6 +406,21 @@ protected function createRelationSerializationContext(string $resourceClass, arr
         return $context;
     }
 
+    /**
+     * Is normalization skipping enabled for this attribute ?
+     *
+     * @param $propertyMetadata
+     * @param array $context
+     *
+     * @return bool
+     */
+    private function isSkippedSerializationAttribute($propertyMetadata, array $context)
```

Comment:
`isPropertySerializable`

Also missing typehints and you can remove the block comments (`@param/@return`):

```
function isPropertySerializable(PropertyMetadata $propertyMetadata, array $context): bool 
{}
```
Then negate: `!$this->isPropertySerializable(...)`